discovery-provider/src/tasks/index_spl_token.py

```diff
@@ -414,15 +424,18 @@ def parse_sol_tx_batch(
     return (update_user_ids, updated_root_accounts, updated_token_accounts)
 
 
-def split_list(list, n):
-    for i in range(0, len(list), n):
-        yield list[i : i + n]
+T = TypeVar("T")
+
```

Comment:
nit: remove one of the empty lines?

Reply:
ask the linter to change its ways 8)  this is 100% autoformat

how about we bring up discussion of what spacing rules we want after this goes in?